utils.py

```diff
@@ -329,7 +330,7 @@ def convert_png_binary_to_data_url(content: Union[str, bytes]) -> str:
     if imghdr.what(None, h=content) == 'png':
         return '%s%s' % (
             PNG_DATA_URL_PREFIX,
-            python_utils.url_quote(base64.b64encode(content))  # type: ignore[no-untyped-call]
+            urllib.quote(base64.b64encode(content))  # type: ignore[no-untyped-call]
```

Comment:
You can do `return '%s%s' % (PNG_DATA_URL_PREFIX, urllib.quote(base64.b64encode(content)))` here.

Reply:
should i remove the comment ```# type: ignore[no-untyped-call]```?

Reply:
Yep, you can.

Reply:
@vojtechjelinek  Done